distribution/scripting.md

```diff
@@ -45,16 +34,74 @@ registerPlugin({
     type: 'remote',
     licence: 'MIT',
     targetApiVersion: 34,
+    minApiVersion: 10,
     main: main
 });
 ```
 
 This will log a message to the terminal screen (`stdout`) when you open any park. If you are on Windows, make sure to run `openrct2.com` instead of `openrct2.exe` so you can interact with the `stdin` / `stdout` console. The console is a JavaScript interpreter (REPL), this means you can write and test expressions similar to the console found in web browsers when you press `F12`. When you make changes to your script, you must exit your current game and open it again for the script to reload... unless you use the hot reload feature.
 
-The hot reload feature can be enabled by editing your `config.ini` file and setting `enable_hot_reloading` to `true` under `[plugin]`. When this is enabled, the game will auto-reload the script in real-time whenever you save your JavaScript file. This allows rapid development of plug-ins as you can write code and quickly preview your changes, such as closing and opening a specific custom window on startup. A demonstration of this can be found on YouTube: [OpenRCT2 plugin hot-reload demo](https://www.youtube.com/watch?v=jmjWzEhmDjk)
+### Script types
+
+There are three types of scripts that can influence how the game loads the script:
+
+* Local
+* Remote
+* Intransient
+
+#### Singleplayer
+
+In singleplayer all plugin types are unrestricted. Local and remote plugins reload each time a scenario or a save file is opened. Plugins of both types can alter game state directly or execute either built-in game actions or custom game actions.
+
+Intransient plugins load when the game starts and stays active until the game is shutdown.
+
+#### Multiplayer
+
+In multiplayer local and remote plugins are different in when they are loaded and how they can alter the game state.
+
+**Local plugins** will load on any client in multiplayer, which will allow each player to bring any local script for their own game without other players in a multiplayer server also needing to also enable the same script.
```

Comment:
What is the misleading part? Also not sure what "as usual" refers too? Singleplayer? That may be misleading because local plugins are not completely "as usual" as singleplayer.

I'll fix the double "also" though, bad copy paste mistake. 🙂

Reply:
"Local plugins will load on any client in multiplayer" can be interpreted as any client that joins will load your local script.
Yes, with usual I meant compared to singleplayer.

Reply:
Good point. Added a few extra words that will hopefully clarify that confusion. 🙂